src/main.rs

```diff
@@ -68,15 +68,19 @@ struct Cli {
     path: Option<PathBuf>,
 
     /// Export the presentation as a PDF rather than displaying it.
-    #[clap(short, long)]
+    #[clap(short, long, group = "export")]
     export_pdf: bool,
 
+    /// Export the presentation as a HTML rather than displaying it.
+    #[clap(short, long, group = "export")]
```

Comment:
This is going to fail because both options start with an `e`. You can drop the `short` option here for now.